### DISCUSSION TITLE
Github account suspended, no mail, etc.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#77840] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I was wondering if anyone had an issue with github suspending their account for no reason.

my account was https://github.com/sirlavoie

i contacted support towards this, no answer.

they never contacted me about my ban

i had my website hosted on vercel via github, I can no longer modify it, lost my school that are due.

does anyone has a fix for this?

**Comment by queenofcorgis** [comment, marked as answer]
Hi @rabecoa 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Reply by ghost** [reply]
Why did I never get a mail concerning my ban tho?

i just logged in and it was over…
